src/core/loaders/Loader.ts

```diff
@@ -200,6 +202,27 @@ export abstract class Loader extends Disposable {
     return str
   }
 
+  private treeNodeValueHasPluralizationKeys(value: Record<string, any>, locale: string) {
+    if (!value)
+      return false
+
+    return isObject(value)
+      && this.pluralizationKeysV4.some((key: string) => get(value, key) != null)
+  }
+
+  private firstPluralizationKeyValue(value: Record<string, any>, locale: string) {
```

Comment:
Do we need both these methods, or would the second suffice? If it returns `undefined` when the value isn't an object or when no pluralization subkey exists, that's probably good enough for our purposes and saves the complexity of writing similar logic twice.

Reply:
Yeah, can probably merge/tidy these up ... just the naming can become messy ... `firstPluralizationKeyValueIfAvailableAndIfTreeNodeIsObject`?! 😉